training/04_editing_eml.Rmd

```diff
@@ -20,6 +24,11 @@ This chapter is a practical tutorial for using R to read, edit, write, and valid
 ```{r, child = '../workflows/edit_eml/edit_otherentities.Rmd'}
 ```
 
+## Exercise 3a {.exercise}
+
+* Make sure your package from [before](#exercise-2) is loaded into R.
+* Replace the existing `dataTable` with a new `dataTable` object with an `attributelist` and `physical` section you write in R using the above commands.
```

Comment:
are we suggesting that people update their package on the node before proceeding to 3b? If so we should be explicit about that

Reply:
hmmm I was thinking that 3a would be focusing on updating datatable and physical in this section and 3b is a continuation where you would be using what you made earlier to update the package... I've pushed a change that hopefully makes the language a bit more clear

also did not know that you can suggest changes... so cool!

Reply:
I know, its one of the reasons I love the PR model of making changes